recipe/meta.yaml

```diff
@@ -12,20 +12,23 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  # ppc64le, s390x and win are all unsupported by jax/jaxlib
+  skip: true  # [ppc64le or s390x or win or py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.6
+    - python
     - absl-py >=0.7.1
-    - chex >=0.0.4
+    - chex >=0.1.5
     - jax >=0.1.55
     - jaxlib >=0.1.37
-    - numpy >=1.18.0
+    - numpy >=1.18.0,<2 
```

Comment:
Do we want to use the same `numpy` requirements as for `jax`/`jaxlib`/`chex` here?

Reply:
Yeah I think that probably makes the most sense. I've updated that. Thanks!